RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+> [***] High Priority, e.g., Major Enhancements, Widespread Crash Fixes, etc.
+> [**] Medium Priority, e.g., Minor Enhancements Users Will Notice, etc.
+> [*] Low Priority, e.g., Minor Enhancements Users May Miss, etc.
```

Comment:
Unfortunately, we'll need to update the formatting here. Release toolkit [expects](https://github.com/wordpress-mobile/release-toolkit/blob/726c93f155900bf04d79c2821dbd11c05fded54d/lib/fastlane/plugin/wpmreleasetoolkit/helper/release_notes_helper.rb#L12-L14) header information such as this to start either with `***` ([like Woo does](https://github.com/woocommerce/woocommerce-ios/blob/518046c4f2ac652ef6f088c9442eb2a4202afc34/RELEASE-NOTES.txt)) or `//`.

```suggestion
// [***] High Priority, e.g., Major Enhancements, Widespread Crash Fixes, etc.
// [**] Medium Priority, e.g., Minor Enhancements Users Will Notice, etc.
// [*] Low Priority, e.g., Minor Enhancements Users May Miss, etc.
```

I verified this using a test branch. See [effect on formatting using `>`](https://github.com/wordpress-mobile/WordPress-iOS/commit/d7c9655ae0637f3a5d66d162db29e6de897eef20) where the new version ends up on top of this info, and [effect with `//`](https://github.com/wordpress-mobile/WordPress-iOS/commit/5695a5491bd75ebcbca0cff1fff7fe72f6e284c2) which behaves as desired.

Reply:
While I'm suggesting changes, what do you think of adding an explanation for that information?

Taking inspiration from [the wording Woo uses](https://github.com/woocommerce/woocommerce-ios/blob/518046c4f2ac652ef6f088c9442eb2a4202afc34/RELEASE-NOTES.txt):

```suggestion
// Please use the following: [<priority indicator>] <description> [<PR URL>]. 
//
// When assigning priorities follow this convention:
//
// - [***] High Priority, e.g., Major Enhancements, Widespread Crash Fixes, etc.
// - [**] Medium Priority, e.g., Minor Enhancements Users Will Notice, etc.
// - [*] Low Priority, e.g., Minor Enhancements Users May Miss, etc.
```